CMakeLists.txt

```diff
@@ -80,8 +80,12 @@ endif()
 
 add_subdirectory(media_driver)
 
+message(STATUS "LIBVA_DRIVERS_PATH = ${LIBVA_DRIVERS_PATH}")
+
+set(LIBVA_DRIVERS_PATH "/usr/lib/x86_64-linux-gnu/dri/")
```

Comment:
You can not hardcode Debian/Ubuntu specific path, this will break other distributions and user setups if user has drivers in a different location.

Reply:
Sounds good, will revert